couchbase/src/main/java/org/hibernate/ogm/datastore/couchbase/logging/impl/LoggerFactory.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.couchbase.logging.impl;
+import org.jboss.logging.Logger;
+
+/**
+ * @author Stawicka Ewa
+ */
+public class LoggerFactory {
+	public static Log getLogger() {
+		return Logger.getMessageLogger( Log.class, "CouchBase" );
```

Comment:
Better pass the caller class name as a category, that's what we do for (most of) the other backends. E.g. check out the logger factory in core.